blossom/settings/base.py

```diff
@@ -17,7 +17,7 @@
 import dotenv
 from django.urls import reverse_lazy
 
-from blossom import __version__
+from blossom import __version__  # noqa
```

Comment:
This should probably be changed to suppress only a specific warning instead of all of them

Reply:
I don't even remember why that's there. Flake8 isn't throwing an error there, so I removed it